pygame/load.py

```diff
@@ -0,0 +1,45 @@
+# dvd logo simulator in python
```

Comment:
Missing shebang

Reply:
It's unclear what Python version this is meant to run on without the shebang line.